tests/unit/advanced-caching.test.js

```
@@ -0,0 +1,1213 @@
+      const response2 = await handleAPIRequest(mockRequest);
+      expect(response2).toBeDefined(); // Should get cached response
+    });
+    
+    test.skip('should implement stale-while-revalidate for gallery API', async () => {
+      const handleGalleryAPIRequest = async (request) => {
+        const cache = await caches.open('alocubano-api-v2.0.0');
+        const cachedResponse = await cache.match(request);
+        
+        // Start background update
+        const networkUpdate = fetch(request).then(async (networkResponse) => {
+          if (networkResponse && networkResponse.ok) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)